multimodal/src/autogluon/multimodal/models/huggingface_text.py

```diff
@@ -28,6 +28,15 @@
 logger = logging.getLogger(AUTOMM)
 
 
+class DummyLayer(nn.Module):
```

Comment:
Better to move `DummyLayer` into utils.py. Need to keep this file clean.

Reply:
We can consider to refactor it later. This is  a issue I met when finetuning T5 models but haven’t seen such issue in other backbones. We may remove the hack later.

Reply:
We don't when Pytorch will fix this. Moving it to utils is still better. This may not be a good example for future developers to follow.

Reply:
If I do not add the DummyLayer, finetuning mT5-XL on MRPC with the implementation can only get 0.70 ACC. As a reference, adding the dummy layer can get 0.872 ACC.

```
# Without Dummy
{'acc': 0.7009803921568627, 'f1': 0.8195266272189349, 'time_cost': 4389.08921289444}` 

# With Dummy
{"acc": 0.8725490196078431, "f1": 0.9047619047619048, "time_cost": 8712.896693944931}
```

Also, we need to wrap dummy layer after the embedding layer so it is not generally applicable.